# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../gtkpod/gtkpod-2.0.1-no-update-icon-cache.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- gtkpod-2.0.1/plugins/media_player/icons/Makefile.am.orig	2011-06-02 02:31:13.236004954 +0200
+++ gtkpod-2.0.1/plugins/media_player/icons/Makefile.am	2011-06-02 02:40:32.512004949 +0200
@@ -5,10 +5,5 @@
 install-data-hook: update-icon-cache
 uninstall-hook: update-icon-cache
 update-icon-cache:
-	@-if test -z "$(DESTDIR)"; then \
-		echo "Updating Gtk icon cache."; \
-		$(gtk_update_icon_cache); \
-	else \
-		echo "*** Icon cache not updated.  After (un)install, run this:"; \
-		echo "***   $(gtk_update_icon_cache)"; \
-	fi
\ No newline at end of file
+	@-echo "*** Icon cache not updated.  After (un)install, run this:"; \
+	  echo "***   $(gtk_update_icon_cache)";
--- gtkpod-2.0.1/plugins/cover_display/icons/Makefile.am.orig	2011-06-02 02:31:13.272004958 +0200
+++ gtkpod-2.0.1/plugins/cover_display/icons/Makefile.am	2011-06-02 02:40:42.520004948 +0200
@@ -5,10 +5,5 @@
 install-data-hook: update-icon-cache
 uninstall-hook: update-icon-cache
 update-icon-cache:
-	@-if test -z "$(DESTDIR)"; then \
-		echo "Updating Gtk icon cache."; \
-		$(gtk_update_icon_cache); \
-	else \
-		echo "*** Icon cache not updated.  After (un)install, run this:"; \
-		echo "***   $(gtk_update_icon_cache)"; \
-	fi
\ No newline at end of file
+	@-echo "*** Icon cache not updated.  After (un)install, run this:"; \
+	  echo "***   $(gtk_update_icon_cache)";
--- gtkpod-2.0.1/plugins/playlist_display/icons/Makefile.am.orig	2011-06-02 02:31:13.304004955 +0200
+++ gtkpod-2.0.1/plugins/playlist_display/icons/Makefile.am	2011-06-02 02:40:47.500004949 +0200
@@ -5,10 +5,5 @@
 install-data-hook: update-icon-cache
 uninstall-hook: update-icon-cache
 update-icon-cache:
-	@-if test -z "$(DESTDIR)"; then \
-		echo "Updating Gtk icon cache."; \
-		$(gtk_update_icon_cache); \
-	else \
-		echo "*** Icon cache not updated.  After (un)install, run this:"; \
-		echo "***   $(gtk_update_icon_cache)"; \
-	fi
\ No newline at end of file
+	@-echo "*** Icon cache not updated.  After (un)install, run this:"; \
+	  echo "***   $(gtk_update_icon_cache)";
--- gtkpod-2.0.1/plugins/core_preferences/icons/Makefile.am.orig	2011-06-02 02:31:13.344004942 +0200
+++ gtkpod-2.0.1/plugins/core_preferences/icons/Makefile.am	2011-06-02 02:39:31.636004948 +0200
@@ -5,10 +5,5 @@
 install-data-hook: update-icon-cache
 uninstall-hook: update-icon-cache
 update-icon-cache:
-	@-if test -z "$(DESTDIR)"; then \
-		echo "Updating Gtk icon cache."; \
-		$(gtk_update_icon_cache); \
-	else \
-		echo "*** Icon cache not updated.  After (un)install, run this:"; \
-		echo "***   $(gtk_update_icon_cache)"; \
-	fi
\ No newline at end of file
+	@-echo "*** Icon cache not updated.  After (un)install, run this:"; \
+	  echo "***   $(gtk_update_icon_cache)";
--- gtkpod-2.0.1/plugins/sorttab_display/icons/Makefile.am.orig	2011-06-02 02:31:13.376004946 +0200
+++ gtkpod-2.0.1/plugins/sorttab_display/icons/Makefile.am	2011-06-02 02:39:38.228004948 +0200
@@ -5,10 +5,5 @@
 install-data-hook: update-icon-cache
 uninstall-hook: update-icon-cache
 update-icon-cache:
-	@-if test -z "$(DESTDIR)"; then \
-		echo "Updating Gtk icon cache."; \
-		$(gtk_update_icon_cache); \
-	else \
-		echo "*** Icon cache not updated.  After (un)install, run this:"; \
-		echo "***   $(gtk_update_icon_cache)"; \
-	fi
\ No newline at end of file
+	@-echo "*** Icon cache not updated.  After (un)install, run this:"; \
+	  echo "***   $(gtk_update_icon_cache)";
--- gtkpod-2.0.1/plugins/photo_editor/icons/Makefile.am.orig	2011-06-02 02:31:13.420004949 +0200
+++ gtkpod-2.0.1/plugins/photo_editor/icons/Makefile.am	2011-06-02 02:39:43.076004949 +0200
@@ -5,10 +5,5 @@
 install-data-hook: update-icon-cache
 uninstall-hook: update-icon-cache
 update-icon-cache:
-	@-if test -z "$(DESTDIR)"; then \
-		echo "Updating Gtk icon cache."; \
-		$(gtk_update_icon_cache); \
-	else \
-		echo "*** Icon cache not updated.  After (un)install, run this:"; \
-		echo "***   $(gtk_update_icon_cache)"; \
-	fi
\ No newline at end of file
+	@-echo "*** Icon cache not updated.  After (un)install, run this:"; \
+	  echo "***   $(gtk_update_icon_cache)";
--- gtkpod-2.0.1/plugins/track_display/icons/Makefile.am.orig	2011-06-02 02:31:13.432004957 +0200
+++ gtkpod-2.0.1/plugins/track_display/icons/Makefile.am	2011-06-02 02:39:48.312004948 +0200
@@ -5,10 +5,5 @@
 install-data-hook: update-icon-cache
 uninstall-hook: update-icon-cache
 update-icon-cache:
-	@-if test -z "$(DESTDIR)"; then \
-		echo "Updating Gtk icon cache."; \
-		$(gtk_update_icon_cache); \
-	else \
-		echo "*** Icon cache not updated.  After (un)install, run this:"; \
-		echo "***   $(gtk_update_icon_cache)"; \
-	fi
\ No newline at end of file
+	@-echo "*** Icon cache not updated.  After (un)install, run this:"; \
+	  echo "***   $(gtk_update_icon_cache)";
--- gtkpod-2.0.1/plugins/coverweb/icons/Makefile.am.orig	2011-06-02 02:31:13.476004939 +0200
+++ gtkpod-2.0.1/plugins/coverweb/icons/Makefile.am	2011-06-02 02:39:53.720004948 +0200
@@ -5,10 +5,5 @@
 install-data-hook: update-icon-cache
 uninstall-hook: update-icon-cache
 update-icon-cache:
-	@-if test -z "$(DESTDIR)"; then \
-		echo "Updating Gtk icon cache."; \
-		$(gtk_update_icon_cache); \
-	else \
-		echo "*** Icon cache not updated.  After (un)install, run this:"; \
-		echo "***   $(gtk_update_icon_cache)"; \
-	fi
\ No newline at end of file
+	@-echo "*** Icon cache not updated.  After (un)install, run this:"; \
+	  echo "***   $(gtk_update_icon_cache)";
--- gtkpod-2.0.1/plugins/mserv/icons/Makefile.am.orig	2011-06-02 02:31:13.504004944 +0200
+++ gtkpod-2.0.1/plugins/mserv/icons/Makefile.am	2011-06-02 02:39:58.672004949 +0200
@@ -5,10 +5,5 @@
 install-data-hook: update-icon-cache
 uninstall-hook: update-icon-cache
 update-icon-cache:
-	@-if test -z "$(DESTDIR)"; then \
-		echo "Updating Gtk icon cache."; \
-		$(gtk_update_icon_cache); \
-	else \
-		echo "*** Icon cache not updated.  After (un)install, run this:"; \
-		echo "***   $(gtk_update_icon_cache)"; \
-	fi
\ No newline at end of file
+	@-echo "*** Icon cache not updated.  After (un)install, run this:"; \
+	  echo "***   $(gtk_update_icon_cache)";
--- gtkpod-2.0.1/icons/Makefile.am.orig	2011-06-02 02:31:13.548004957 +0200
+++ gtkpod-2.0.1/icons/Makefile.am	2011-06-02 02:40:03.220004947 +0200
@@ -10,10 +10,5 @@
 install-data-hook: update-icon-cache
 uninstall-hook: update-icon-cache
 update-icon-cache:
-	@-if test -z "$(DESTDIR)"; then \
-		echo "Updating Gtk icon cache."; \
-		$(gtk_update_icon_cache); \
-	else \
-		echo "*** Icon cache not updated.  After (un)install, run this:"; \
-		echo "***   $(gtk_update_icon_cache)"; \
-	fi
+	@-echo "*** Icon cache not updated.  After (un)install, run this:"; \
+	  echo "***   $(gtk_update_icon_cache)";
